src/agent/service.ts

```diff
@@ -51,7 +51,8 @@ if (typeof svcinstall[action] !== 'function') {
 }*/
 
 // node is known as nodejs on some *nix installs
-var nodePath = shelljs.which('nodejs') || shelljs.which('node');
+//var nodePath = shelljs.which('nodejs') || shelljs.which('node');
+var nodePath = path.resolve('__dirname/..', 'runtime/node/bin/node')
```

Comment:
I would probably also do a fileExistsSync check and make sure it fails early with a good message (in the event someone is using direct install from npm out of habit).  The error message should say node is missing, see latest docs on how to install the agent.